api/contract.go

```diff
@@ -75,6 +75,10 @@ type createFilterOutputRequest struct {
 	model.FilterOutput
 }
 
+type eventRequest struct {
```

Comment:
Please specify create/delete/get etc for contract structs

Reply:
done